spring-xd-modules.tex

```diff
@@ -1,44 +1,45 @@
 \section{Modules}
 \label{sec:Modules}
-A Spring XD Module \cite{modules} is a unit of data processing and as mentioned above 
+A Spring XD Module \cite{modules} is a data processing unit and as mentioned above
 there are four types: source, processor, sink, and job. Modules in Spring XD are defined 
 in their own application context. This allows for easy encapsulation and life cycle 
 management for modules. Additionally, the use of an application context allows for easy 
 module expansion.  Spring XD uses Spring Integration \cite{spring-integration-reference} 
-as its foundation for implementing modules. A module is comprised of one or more channels 
-and an integration bean.
+as its foundation for implementing modules. A module is comprised of component that
+implements the data processing logic and one or more connectors(known as channels)
+that connect to the underlying message bus.
 
 \par
 
 Spring XD offers a suite of 23 sources, 24 sinks, 9 processors and 9 jobs that are ready 
-to use at startup.  These modules integrate with a variety of well
-known and popular data stores and processing systems such as JDBC, HDFS, MongoDB, Spark,
-and Sqoop.  If an existing module does not meet the needs of a given use case, Spring XD
-supports custom modules.
+to use at startup.  These modules integrate with a variety of well known and popular
+data stores and processing systems such as JDBC, HDFS, MongoDB, Spark, Kafka, RabbitMQ,
+Sqoop etc.,  If an existing module does not meet the needs of a given use case, Spring XD
+supports developing custom modules.
```

Comment:
Isn't it implied that custom modules require development? I'm not sure the addition of "developing" is needed here.